examples/seek_viewer.cpp

```diff
@@ -164,17 +183,54 @@ int main(int argc, char** argv)
         }
 
         // Retrieve frame from seek and process
-        process_frame(seekframe, outframe, scale, colormap, rotate);
+        process_frame(seekframe, outframe, scale, colormap, rotate, normalize, brightness);
 
         if (output == "window") {
             imshow("SeekThermal", outframe);
-            char c = waitKey(10);
-            if (c == 's') {
+		namedWindow("SeekThermal");
+		moveWindow("SeekThermal", x,y);
+            int c = waitKeyEx(5);
+		//std::cout << (int)c << std::endl;
+            if (c == 112) {
```

Comment:
Processing these events should be handled in a dedicated function to separate out those concerns.